Reflector/Reflector.Src/Reflector.cs

```diff
@@ -0,0 +1,115 @@
+using System.Reflection;
+using System.Text;
+
+namespace Reflector;
+
+/// <summary>
+/// Reflector class to print or compare classes
+/// </summary>
+public class ReflectorClass
+{
+    /// <summary>
+    /// Print class in <class-name>.cs file
+    /// </summary>
+    /// <param name="someClass">Class to print</param>
+    public static void PrintStructure(Type someClass)
+    {
+        using StreamWriter outputFile = new StreamWriter(someClass.Name.Split('.').Last() + ".cs");

```

Comment:
```suggestion
        using var outputFile = new StreamWriter(someClass.Name.Split('.').Last() + ".cs");
```
И дальше фигурных скобок не надо